src/main/java/com/sda/microblogging/service/FollowerService.java

```diff
@@ -53,6 +53,10 @@ public void unFollowUser(Integer userId, Integer followerId) {
         }
     }
 
+    public int countFollowersByUserId (Integer userId){
+        return getAllFollowersByUserId(userId).size();
+    }
```

Comment:
You can use count query instead of getting the whole user objects

Reply:
of course, but can it be postponed?